ezyfox-server-nio/src/main/java/com/tvd12/ezyfoxserver/nio/EzySocketServerBootstrap.java

```diff
@@ -122,36 +142,25 @@ private ServerSocketChannel newServerSocketChannel() throws Exception {
         return ServerSocketChannel.open();
     }
 
-    private int getSocketReaderPoolSize() {
+    private int getSocketReaderThreadPoolSize() {
         return EzyNioThreadPoolSizes.SOCKET_READER;
     }
 
-    private int getSocketWriterPoolSize() {
-        return getSocketSetting().getWriterThreadPoolSize();
-    }
-
-    private int getSocketAcceptorPoolSize() {
+    private int getSocketAcceptorThreadPoolSize() {
         return EzyNioThreadPoolSizes.SOCKET_ACCEPTOR;
     }
 
-    private int getSocketPort() {
-        return getSocketSetting().getPort();
-    }
-
-    private String getSocketAddress() {
-        return getSocketSetting().getAddress();
-    }
-
-    private boolean getSocketTcpNoDelay() {
-        return getSocketSetting().isTcpNoDelay();
-    }
+    public static class Builder extends EzyAbstractSocketServerBootstrap.Builder<
+        Builder,
+        EzySocketServerBootstrap
+        > {
```

Comment:
This might be one level less of indentation ?

```java
    public static class Builder extends EzyAbstractSocketServerBootstrap.Builder<
        Builder,
        EzySocketServerBootstrap
    > {
```

looks more consistent with our convention:

```java
public void methodName(
    TypeA paramA,
    TypeB paramB
) {  <---
```

Reply:
Cause by check style error

```
ezyfox-server/ezyfox-server-nio/src/main/java/com/tvd12/ezyfoxserver/nio/EzySocketServerBootstrap.java:156:5: '>' has incorrect indentation level 4, expected level should be 8. [Indentation]
```